en_CA/webapp/src/main/webapp/i18n/all_en_CA.properties

```diff
@@ -853,8 +853,7 @@ reordering_menus_failed = Reordering of menu items failed.
 page = page
 view_page = View page
 of_the_results = of the results
-there_are_no = There are no
-individuals_names_starting_with = individuals whose name starts with
+there_are_no_entry_starting_with = There are no entry starting with
```

Comment:
Should this be either "There are no entries..."? or "There is no entry..."?

Reply:
https://jira.lyrasis.org/secure/attachment/20055/JIRA%201870%20b.png

Reply:
Should I update?

Reply:
Probably. I think the grammar is wrong.

Reply:
Will update.